src/java.sql.rowset/share/classes/com/sun/rowset/RowSetResourceBundle.properties

```diff
@@ -123,15 +123,15 @@ crsreader.caldetected = Detected a Calendar
 #CachedRowSetWriter exceptions
 crswriter.connect = Unable to get connection
 crswriter.tname = writeData cannot determine table name
-crswriter.params1 = Value of params1 : {0} 
-crswriter.params2 = Value of params2 : {0} 
-crswriter.conflictsno =  conflicts while synchronizing 
+crswriter.params1 = Value of params1 : {0}\u0020
+crswriter.params2 = Value of params2 : {0}\u0020
+crswriter.conflictsno =  conflicts while synchronizing\u0020
 
 #InsertRow exceptions
 insertrow.novalue = No value has been inserted
 
 #SyncResolverImpl exceptions
-syncrsimpl.indexval = Index value out of range  
+syncrsimpl.indexval = Index value out of range\u0020\u0020
```

Comment:
prob. unnecessary

Reply:
This case is similar the one you mentioned below. If this value is used in a string template or concatenation, the trailing white-space could be necessary; however, one space is probably enough.